actors/test/committed_capacity_scenario_test.go

```diff
@@ -1,4 +1,4 @@
-package test_test
+package test
```

Comment:
🎉 

But also, some of this noise would have been better left for a separate PR.